### ISSUE TITLE
BCUS98152 - SOCOM: U.S. Navy SEALs Confrontation

### ISSUE THREAD

**Issue by DeadOnAim** [RPCS3/rpcs3#14229]
### Quick summary

Was able to load into lobby but upon joining game I got to where players were playing and froze instantly, it doesn't happen all the time, it seems random.

### Attach a log file

[RPCS3.zip](https://github.com/RPCS3/rpcs3/files/12100296/RPCS3.zip)

### System configuration

Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz | 8 Threads | 15.94 GiB RAM | TSC: 4.008GHz | AVX+ | FMA3 | TSX
Operating system: Windows, Major: 10, Minor: 0, Build: 19045, Service Pack: none, Compatibility mode: 0
Qt version: Compiled against Qt 5.15.2 | Run-time uses Qt 5.15.2

**Comment by Asinin3** [comment]
**Bad Settings**
Default Resolution was changed from the default & recommended 1280x720 - Why? The game wont be 1080p internal so you should keep it at 1280x720 and use Resolution Scaling at 200% (better than 150% due to clean integer) instead. 1920X1080 Internal Resolution setting can also cause issues in many games. We typically only recommend it for older PS2 games that were ported to PS3 or easy to run PSN games e.g side-scrollers that are confirmed to run at 1920x1080 internally. (There are only a few exceptions).

Also description is missing info, you talk about a lobby so is this with you trying to run it through netplay/rpcn or what?

**Comment by Sapelain** [comment]
Hi, you can stop by and look at the post through the settings

https://github.com/RPCS3/rpcs3/issues/14242

Although there is a small failure the game works perfect and does not close

**Comment by Asinin3** [comment]
No, he only needs to change the resolution setting I mentioned most-likely.

**Comment by DeadOnAim** [comment]
I will make the adjustments recommend and do more testing then respond thank you.

**Comment by DeadOnAim** [comment]
[RPCS3.log.zip](https://github.com/RPCS3/rpcs3/files/12139144/RPCS3.zip)

Made the recommended changes and I am freezing more often was joining a game and froze then was able to get in a game play 2 rounds and when the game went to switch rounds I froze upon loading into the third round of gameplay.

**Comment by DeadOnAim** [comment]
[RPCS3.log.zip](https://github.com/RPCS3/rpcs3/files/12140205/RPCS3.log.zip)
Game crashed again with same settings while entering spectator mode.

**Comment by Sapelain** [comment]
It is an error entered in the last updates. Besides me, the audio is getting stuck now, I don't know if that affects others too.
When you see him come out, go in to edit weapons. If there is luck.

Other games will have improved but this one is breaking him.

**Comment by DeadOnAim** [comment]
[RPCS3.zip](https://github.com/RPCS3/rpcs3/files/12178386/RPCS3.zip)
Game seems to keep crashing when entering spectator mode alot more frequently with these settings.

**Comment by digant73** [comment]
@DeadOnAim is that still an issue?

**Comment by ScottNash042** [comment]
Can be closed
Original poster hasn't updated in over 3 years

**Comment by DeadOnAim** [comment]
yes there are alot of issues with this title -- no one was responding but there is a community trying to play.
